website/src/_includes/section-menu/documentation.html

```diff
@@ -117,4 +117,5 @@
     <li><a href="{{ site.baseurl }}/documentation/resources/learning-resources/#how-to-contribute">How to Contribute</a></li>
     <li><a href="{{ site.baseurl }}/documentation/resources/videos-and-podcasts">Videos and Podcasts</a></li>
   </ul>
+  <li><a href="https://cwiki.apache.org/confluence/display/BEAM/Apache+Beam">Beam Wiki</a></li>
 </li>
```

Comment:
staging is failing now, I think it's this `</li>` needs to be moved above the new wiki addition

Reply:
Fixed it, thanks